scripts/templates/docker-compose.template

```diff
@@ -36,6 +36,7 @@ hydroshare:
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
     SOLR_HOST: solr
+    TMP: /hs_tmp
```

Comment:
@aphelionz Yes, either auto-generated based on docker-compose.template if USE_LOCAL_IRODS setting is false, or based on docker-compose-local-irods.template if USE_LOCAL_IRODS setting is true. @mjstealey can provide more details if needed.